static/js/main.js

```diff
@@ -51,6 +51,7 @@ function buildPage() {
     }
     completeTypeData();
     completePokemonData();
+    slugs = idToName( slugs );
```

Comment:
Probably redundant, but I never know if I'm passing to a function 'a copy of' or 'a pointer to'. JS' permissiveness will always confuse my c brain but whatever it just works :')